src/ProgramChangeController.js

```diff
@@ -28,9 +28,9 @@ export default class ProgramChangeController {
             if (selectedAction) {
                 this.playAnnouncementForAdd(selectedAction);
                 this.doActivitiesForAdd(programBlockEditor, index);
+
```

Comment:
Please remove the changes to `ProgramChangeController` as they are just layout changes.

Reply:
Done.